packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -105,22 +105,21 @@ data "template_file" "controller_host_endpoints" {
 
 data "template_file" "worker_host_endpoints" {
   count    = "${var.worker_count}"
-  template = "${file("${path.module}/calico/worker_host_endpoints.yaml.tmpl")}"
+  template = "${file("${path.module}/calico/worker-host-endpoint.yaml.tmpl")}"
 
   vars {
     node_name = "${element("${var.worker_nodes_hostnames}", count.index)}"
   }
 }
 
 data "template_file" "host_protection_policy" {
-  template = "${file("${path.module}/calico/calico-policy.yaml.tmpl")}"
+  template = "${file("${path.module}/calico/host-protection.yaml.tmpl")}"
 
   vars = {
     controller_host_endpoints = "${join("\n", data.template_file.controller_host_endpoints.*.rendered)}"
     worker_host_endpoints     = "${join("\n", data.template_file.worker_host_endpoints.*.rendered)}"
     management_cidrs          = "${jsonencode("${var.management_cidrs}")}"
     cluster_internal_cidrs    = "${jsonencode(list("${var.node_private_cidr}", "${var.pod_cidr}", "${var.service_cidr}"))}"
     etcd_server_cidrs         = "${jsonencode("${packet_device.controllers.*.access_private_ipv4}")}"
-    node_public_ips           = "${jsonencode(concat("${packet_device.controllers.*.access_public_ipv4}", "${var.worker_nodes_public_ipv4s}"))}"
```

Comment:
what I mean, now that I see the line again, is that `var.worker_nodes_public_ipv4s` is not used anywhere else in the module, AFAIK, so we *can* remove it or deprecate it or whatever. Having unused variables is not nice :)

Reply:
Ah, I thought you were referring to `node_public_ips`. Yes, not that *this* one is removed there are no other uses for `worker_nodes_public_ipv4s` indeed. Fixed, and thanks for noticing.